packages/ckeditor5-image/src/imageinsert/ui/imageinsertformview.ts

```diff
@@ -49,15 +50,15 @@ export default class ImageInsertFormView extends View {
 	/**
 	 * A collection of the defined integrations for inserting the images.
 	 */
-	private readonly children: ViewCollection;
+	private readonly children: ViewCollection<FocusableView>;
 
 	/**
 	 * Creates a view for the dropdown panel of {@link module:image/imageinsert/imageinsertui~ImageInsertUI}.
 	 *
 	 * @param locale The localization services instance.
 	 * @param integrations An integrations object that contains components (or tokens for components) to be shown in the panel view.
 	 */
-	constructor( locale: Locale, integrations: Array<View> = [] ) {
+	constructor( locale: Locale, integrations: Array<FocusableView> = [] ) {
```

Comment:
Is there a scenario where integrations are not `FocusableView`s? If yes, then maybe we should filter `integrations` when we add them to `this._focusables`?

Reply:
I see that you already applied such filtering in some places in this PR.

Reply:
We require these views to be focusable already on master. My changes only reflect this  

https://github.com/ckeditor/ckeditor5/blob/b5c31df8ca7f2032256513532449f424bbc023dd/packages/ckeditor5-image/src/imageinsert/imageinsertui.ts#L252-L257

https://github.com/ckeditor/ckeditor5/blob/f0a702cf32fa47d245b03512417d89abd7f45da7/packages/ckeditor5-image/src/imageinsert/imageinsertui.ts#L183-L188